src/views/Dashboard.vue

```diff
@@ -11,10 +11,22 @@
       <article id="dashboard-article">
         <v-row class="mt-n9">
           <v-col cols="12" md="9">
+            <!-- Alerts section-->
+            <section v-if="alertCount > 0" id="dashboard-alerts-section">
+              <header>
+                <h2 class="mb-3">
+                  <span>Alerts</span>&nbsp;<span class="gray6">({{ alertCount }})</span>
+                </h2>
+              </header>
+              <!-- FUTURE: move these to a new Alerts component and inside one expansion panel -->
+              <MissingInformation v-if="isMissingInformationAlert"/>
+              <NotInGoodStanding v-if="isNotInGoodStandingAlert"/>
+            </section>
```

Comment:
Looks like this:

![image](https://user-images.githubusercontent.com/10002889/195748744-ccff6717-9c34-4218-8e6e-1ae4e9af8c56.png)